lib/screens/map/widgets/zoom_buttons.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+
+class FlutterMapZoomButtons extends StatelessWidget {
+  final void Function() onPressedZoomIn;
+  final void Function() onPressedZoomOut;
+  final double minZoom;
+  final double maxZoom;
+  final double padding;
+  final Alignment alignment;
+  final bool isMini;
+  final IconData zoomInIcon;
+  final IconData zoomOutIcon;
+  final Color? zoomInColor;
+  final Color? zoomInColorIcon;
+  final Color? zoomOutColor;
+  final Color? zoomOutColorIcon;
+
+  const FlutterMapZoomButtons(
+      {super.key,
+      required this.onPressedZoomIn,
+      required this.onPressedZoomOut,
+      this.minZoom = 4,
```

Comment:
is it a level of zoom ? why is it in the view?

Reply:
It's the default value in case it's not informed. But I'll pass those parameter as required to move them out of the view